go.mod

```diff
@@ -12,7 +12,7 @@ replace (
 
 	// See https://github.com/siderolabs/go-loadbalancer/pull/4
 	// `go get github.com/smira/tcpproxy@combined-fixes`, then copy pseudo-version there
-	inet.af/tcpproxy => github.com/smira/tcpproxy v0.0.0-20201015133617-de5f7797b95b
+	inet.af/tcpproxy => github.com/smira/tcpproxy v0.0.0-20200125044825-b6bb9b5b8252
```

Comment:
i guess this is correct, the one renovate shows is wrong

Reply:
yep, reverted